### ISSUE TITLE
Cannot select option when bind data dynamic into nb-select

### ISSUE THREAD

**Issue by tuannh2** [akveo/nebular#1063]
I'm using nb-select for my project, and when i bind data auto in nb-select, i cannot select option as below images:

![image](https://user-images.githubusercontent.com/33337972/49795169-e596e680-fd6b-11e8-8b0f-448a8fc656c6.png)

![image](https://user-images.githubusercontent.com/33337972/49795213-04957880-fd6c-11e8-80c3-8ec37bee9d80.png)

my code in component:

"
import { Component } from '@angular/core';
import { NbMenuService } from '@nebular/theme';
import { TaxAuthorityService } from '../../@core/services/taxAuthority.service';

@Component({
    selector: 'ngx-organization',
    styleUrls: ['organization.component.scss'],
    templateUrl: 'organization.component.html',
})

export class OrganizationComponent {
    taxAuthorities;

    constructor(private menuService: NbMenuService, private taxAuthorityService: TaxAuthorityService) {
        //this.taxAuthorities = taxAuthorityService.getTaxAuthorities();
    }

    ngOnInit() {
        this.taxAuthorityService.getTaxAuthorities().subscribe(p=>this.taxAuthorities = p);
    }
}
"

Thanks!

Tuan,

**Comment by gaspardip** [comment]
Its already fixed by #1009 but Nebular 3.0.1 is not complete yet

Duplicates: #948 #931

**Comment by tanzeelrana** [comment]
is this fix released ?

**Comment by tuannh2** [comment]
Thanks @tanzeelrana. 

I already used the normal select control instead of nb-select.

Tuan,

**Comment by jaapcrezee** [comment]
This does not seem fixed in Nebular v3.3.0. When I set my .ts property to = null and use  data subscription and
```
  <nb-select *ngIf="items"
      placeholder="{{selector_input.placeholder}}"
  >
    <ng-container *ngFor="let item of items">
      <nb-option>{{ item }}</nb-option>
    </ng-container>
</nb-select>
```
there is no data in the dropdown select; <select/> works as expected.

**Comment by nnixaa** [comment]
@Tibing can you check?

**Comment by aefox** [comment]
the initially raised issue with selecting the option actually works as expected as long as your `nb-option`s correctly set their `value` attribute. please see this [stackblitz](https://stackblitz.com/edit/github-kkt2hz-ugcvqu?file=src%2Fapp%2Fhome%2Fhome.component.html) for examples. If you remove the `[value]="item"` on the `nb-option`s you'll see the problem you mentioned reproduces. but this is an incorrect usage of `nb-option`. cc: @nnixaa maybe this needs better documentation?

In the second case when using an `ng-container` as wrapper for `nb-option`s the problem is indeed reproducible on nebular 3.1.0 and 3.3.0. I already have a fix and currently writing some tests. Will raise PR today

**Comment by aefox** [comment]
[PR](https://github.com/akveo/nebular/pull/1280) raised

**Comment by sutra** [comment]
It seems that, this issue can be reproduced in 3.0.0, and fixed in 3.0.1.
